src/classes/OPP_OpportunityAmountSync_SVC.cls

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Please add this to the ApexDocContent/Opportunity.htm files, In a new section called Accounting and Data Consistency